### ISSUE TITLE
Idle time in gerrit show-connections isn't accurate

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evanchueng/gerrit#181]
```
Reported by Shawn Pearce <sop@google.com> on Wed May 20 11:18:19 PDT 2009
Source: JIRA GERRIT-181
Affected Version: 2.0.12

OpenSSH clients periodically send ping messages to the server, to see if the
server is still alive.  These pings are sent over the encrypted channel, and
thus are processed out by the MINA NIO processor threads, which causes the
idle time for the connection to be reset to 0 on the read side, and then again
on the write side when the MINA SSHD event handler actually responds on the
NIO processor thread context.

In other words, the idle time displayed is the time since the client last
pinged us, and isn't the time since the last command completed, or the last
real data transfer occurred.
```

Original issue reported on code.google.com by `code-rev...@gtempaccount.com` on 24 Sep 2009 at 7:39
